<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadioPane</class>
 <widget class="QWidget" name="RadioPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="FavouriteStationsWidget" name="favouriteStations"/>
     <widget class="StationSearchWidget" name="searchPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FavouriteStationsWidget</class>
   <extends>QWidget</extends>
   <header>favouritestationswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StationSearchWidget</class>
   <extends>QWidget</extends>
   <header>stationsearchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
